tests/unit/test_lint.py

```diff
@@ -75,3 +77,48 @@ def test_nclcodestyle():
         """))
 
     assert success, "Your NCL code does not follow our formatting standards."
+
+
+def test_r_lint():
+    """Test R lint."""
+    environ = dict(os.environ)
+    try:
```

Comment:
I recommend using the pytest `monkeypatch` fixture for setting environmental variables: 
https://docs.pytest.org/en/latest/reference.html#_pytest.monkeypatch.MonkeyPatch.setenv

Reply:
Thanks!